### DISCUSSION TITLE
Welcome to Thorium-AVX2 Discussions!

### DISCUSSION THREAD

**Discussion by Alex313031** [Alex313031/Thorium-Win-AVX2#13] [Announcements]
## 👋 Welcome!
  I'm using Discussions as a place to connect with Thorium and Chromium users/devs. Here you can:
  * Ask ANY/ALL questions you’re wondering about. (It doesn't even have to concern Thorium.)
  * Share ideas. (FOSS, Linux, and Thorium ideas are especially welcome.)
  * Engage with other community members.
  * Be respectful and open-minded. Remember that we build the GitHub community and the software it hosts together 💪.
  * Ask about future plans and ask to contribute/make pull requests.

  To get started, comment below with an introduction of yourself and tell us about what you do within the scope of this community or the Thorium browser.

<img src="https://raw.githubusercontent.com/Alex313031/Thorium/main/logos/STAGING/error_dog.png" width="256">

**Comment by kimboslice99** [comment]
Just wanted to say thanks for all the work youve put into this, I switched from edge because they broke HEVC over a year ago and MS is still slacking

I didnt really think it could be much faster, figured thatd be down to my slow starlink connection, but to my surprise, this thing loads like greased lightening!

By the way, I think the image you put in this post has the wrong domain, it should be raw.githubusercontent.com :)

**Reply by Alex313031** [reply]
Thanks, and thanks, updated it.

**Reply by kimboslice99** [reply]
~actually it seems h265 doesnt work for me, canPlayType, isTypeSupported and mediaCapabilities all report that it should, yet chrome://media-internals/#CDMs does not. does this rely upon hardware decoding support or will it fall back to software decoding?~

Nevermind, another jellyfin bug, lol

**Comment by katastrophe92** [comment]
Hey @Alex313031 !! First of all thanks for this best looking Chromium Browser! Just a quick question. I installed the AVX2 version on my i7 gen. 8. Is this the correct installed version? Thanks
